federation/cmd/federation-controller-manager/app/options/options.go

```diff
@@ -118,6 +119,8 @@ func (s *CMServer) AddFlags(fs *pflag.FlagSet) {
 	fs.IntVar(&s.APIServerBurst, "federated-api-burst", s.APIServerBurst, "Burst to use while talking with federation apiserver")
 	fs.StringVar(&s.DnsProvider, "dns-provider", s.DnsProvider, "DNS provider. Valid values are: "+fmt.Sprintf("%q", dnsprovider.RegisteredDnsProviders()))
 	fs.StringVar(&s.DnsConfigFile, "dns-provider-config", s.DnsConfigFile, "Path to config file for configuring DNS provider.")
-	fs.BoolVar(&s.EnableIngressController, "enable-ingress-controller", s.EnableIngressController, "Enable federation ingress controller.")
+	fs.Var(&s.Controllers, "controllers", ""+
+		"A set of key=value pairs that describe controller configuration that may be passed "+
```

Comment:
I wouldnt block this PR on that but will be great if we can list all valid values here. For ex: right now "ingress" is the only valid value. I can see people getting confused by should the key for ingress controller be "ingress" or "Ingress" or "ingress-controller" or "ingressController" or many more such combinations :)